model-testkit/shared/src/main/scala/explore/model/arb/ArbObservation.scala

```diff
@@ -126,7 +131,7 @@ trait ArbObservation:
        Option[Configuration],
        ObservationWorkflow,
        Option[Group.Id],
-       Short
+       (Short, Execution)
```

Comment:
Apparently there is a tuple size limit of 22 here. It took me awhile to figure that out. 😦

Reply:
I got thit by that recently as well